### ISSUE TITLE
QA test tce_c20_triplet has many warnings during ROHF iterations.

### ISSUE THREAD

**Issue by xiongyan21** [nwchemgit/nwchem#145]
QA test tce_c20_triplet has many warnings during ROHF iterations, diffferent from that in the test.
My unifinished test
[c20_cage_triplet_tz.log](https://github.com/nwchemgit/nwchem/files/3555720/c20_cage_triplet_tz.log)

**Comment by edoapra** [comment]
@xiongyan21 Please post convergence problems to the forum.
There are plenty of known workarounds

**Comment by xiongyan21** [comment]
The 4e integrals are too large for my hard disk to handle.

**Comment by edoapra** [comment]
No, that's not the reason for the warnings 
```
#quartets = 9.247D+07 #integrals = 3.285D+09 #direct =  0.0% #cached =100.0%
```
This line shows that 100% of the integrals were written on disk. If you don't want to do it, you might want to focus on reading the manual and use the direct option, instead.
The warning lines are related to symmetry and they are very likely to be harmless

**Comment by xiongyan21** [comment]
Dear Dr. Edoapra
  In fact, I repeated the test yesterday , and got PID 0 error. I cannot remember the error message because I have discarded the log file. According to my experience, this means the disk is full or the I/O is too slow to handle the data produced by CPU.  I just have tried using one thread to do it and find it is very slow, thus I stop it.

**Comment by xiongyan21** [comment]
4-electron integrals stored in orbital form
  available GA memory            4189117680  bytes
 ------------------------------------------------------------------------
 createfile: failed ga_create size/nproc bytes         18991625696
 -----------------------------------------------------------------------
The one thread run using the original memory allocation gives the above.
